src/Parser/DeprecatedHelperVisitor.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace mglaman\PHPStanDrupal\Parser;
+
+use PhpParser\Node;
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitor;
+use PhpParser\NodeVisitorAbstract;
+
+class DeprecatedHelperVisitor extends NodeVisitorAbstract implements NodeVisitor
+{
+
+    private ?Node\Arg $deprecatedCall = null;
+
+    public function enterNode(Node $node)
+    {
+        if ($node instanceof Node\Expr\StaticCall) {
+            if ($node->class instanceof Node\Name\FullyQualified && $node->class->toString() === 'Drupal\Component\Utility\DeprecationHelper') {
+                $this->deprecatedCall = $node->getArgs()[2];
+                return null;
+            }
+        }
+
+        if ($this->deprecatedCall !== null && $node instanceof Node\Arg && $this->isSavedArgument($node)) {
+            $this->deprecatedCall = null;
```

Comment:
The null check is technically duplicated in the `isSavedArgument` method

Reply:
I was making phpstan happy, the property can be null.